yourgov/components/FormMobileFoodVendorPermit/task1/FormTask1Step1.tsx

```diff
@@ -0,0 +1,161 @@
+						<Prose>
+							<p>
+								We’re working hard to improve the way we do business with you.
+								This includes making applications and registrations easier to
+								use.
+							</p>
+							<p>
+								If we already have some of the information you need to tell us,
+								we’ll pre-fill it into your applications. This saves you
+								entering all your details yourself.
+							</p>
+							<p>
+								It’s important to check the pre-filled information in your
+								report before you submit it.
+							</p>
+						</Prose>
+					</Details>
+					{isSuccessMessageVisible && (
+						<SectionAlert
+							focusOnMount
+							title="Business owner details have been updated"
+							tone="success"
+							onClose={() => setIsSuccessMessageVisible(false)}
+						/>
+					)}
+					<Stack gap={1.5} alignItems="flex-start">
+						<SummaryList>
+							<SummaryListItem>
+								<SummaryListItemTerm>First name</SummaryListItemTerm>
+								<SummaryListItemDescription>
+									{stepFormState?.firstName}
+								</SummaryListItemDescription>
+							</SummaryListItem>
+							<SummaryListItem>
+								<SummaryListItemTerm>Last name</SummaryListItemTerm>
+								<SummaryListItemDescription>
+									{stepFormState?.lastName}
+								</SummaryListItemDescription>
+							</SummaryListItem>
+							<SummaryListItem>
+								<SummaryListItemTerm>Email address</SummaryListItemTerm>
+								<SummaryListItemDescription>
+									{stepFormState?.email}
+								</SummaryListItemDescription>
+							</SummaryListItem>
+						</SummaryList>
+						<ButtonLink
+							href={task1FormSteps[0].items && task1FormSteps[0].items[0].href}
+							variant="text"
+						>
+							Change business owner details
+						</ButtonLink>
+					</Stack>
+				</Stack>
+				<AdditionalDetailsForm />
+			</Stack>
+		</FormTask1Container>
+	);
+}
+
+function AdditionalDetailsForm() {
+	const { formState, setFormState, isSavingBeforeExiting } = useGlobalForm();
+	const { submitStep } = useFormTask1Context();
+
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm<Task1Step1Part2FormSchema>({
+		defaultValues: {
+			contactPhoneNumber: formState.task1?.step1?.contactPhoneNumber,
+		},
+		resolver: zodResolver(task1Step1Part2FormSchema),
+		mode: 'onSubmit',
+		reValidateMode: 'onBlur',
+	});
+
+	const onSubmit: SubmitHandler<Task1Step1Part2FormSchema> = async (data) => {
+		if (isSavingBeforeExiting) {
+			return;
+		}
+		await submitStep();
+		setFormState({
+			...formState,
+			task1: {
+				...formState.task1,
+				step1: {
+					...formState.task1?.step1,
+					...data,
+					completed: !isSavingBeforeExiting,
+					started: true,
+				},
+			},
+		});
+	};
+
+	return (
+		<Stack as="form" gap={3} onSubmit={handleSubmit(onSubmit)}>
+			<Stack gap={1.5}>
+				<H2>Additional details</H2>
+				<TextInput
+					label="Contact phone number"
+					hint="Any Australian mobile or landline. For example, 0444111222 or 02 9988 7766"
```

Comment:
Maybe it doesn't matter in this context but that mobile number is two digits too long 😅